src/style/properties.js

```diff
@@ -48,6 +48,7 @@ const styfn = {};
     bgFit: { enums: [ 'none', 'contain', 'cover' ], multiple: true },
     bgCrossOrigin: { enums: [ 'anonymous', 'use-credentials' ], multiple: true },
     bgClip: { enums: [ 'none', 'node' ], multiple: true },
+    bgSmooth: { enums: [ 'yes', 'no' ], multiple: true },
```

Comment:
Should probably be named something like `bools` so that it could be used for other props in future.  See `bool` prop that follows